src/handle.ml

```diff
@@ -134,7 +135,8 @@ let handle_cmd_aux : sig_state -> command -> sig_state * proof_data option =
           fatal x.pos "We have %s ≔ %a." x.elt pp t
         end;
       (* Actually add the symbol to the signature. *)
-      let s = Sign.add_symbol ss.signature Defin x a in
+      let implicits = List.map (fun (_,_,z) -> z) xs in
```

Comment:
Why do you compute implicits from xs and not a?
And why don't you use: let implicits = Scope.get_params_implicitness_of_type a, like in previous case?